test/c_api_tests.cpp

```diff
@@ -120,21 +120,27 @@ TEST(CAPITests, GreedySearchGptFp32CAPI) {
 
   // Verify outputs match expected outputs
   for (int i = 0; i < batch_size; i++) {
-    auto sequence = generator->GetSequence(i);
```

Comment:
Why did this need to change?

Reply:
this file builds with C++17 too. now, GetSequence() is only defined for C++20.